homework7/7.2/7.2/Clock.cs

```diff
@@ -0,0 +1,138 @@
+using System.Runtime.InteropServices;
+
+namespace Task7_2;
+public partial class Clock : Form
+{
+    [DllImport("user32", CharSet = CharSet.Auto)]
+    internal extern static bool PostMessage(IntPtr hWnd, uint Msg, uint WParam, uint LParam);
+
+    [DllImport("user32", CharSet = CharSet.Auto)]
+    internal extern static bool ReleaseCapture();
+
+    /// <summary>
+    /// ���������� ������ ����������
+    /// </summary>
+    private (int x, int y) center = (175, 175);
+
+    /// <summary>
+    /// ����� �������� �������
+    /// </summary>
+    private int secondHandLength = 100;
+
+    /// <summary>
+    /// ����� �������� �������
+    /// </summary>
+    private int minuteHandLength = 90;
+
+    /// <summary>
+    /// ����� ������� �������
+    /// </summary>
+    private int hourHandLength = 75;
+    System.Windows.Forms.Timer clock = new System.Windows.Forms.Timer();
+    public Clock()
```

Comment:
```suggestion
    private int hourHandLength = 75;
    System.Windows.Forms.Timer clock = new System.Windows.Forms.Timer();
    public Clock()
```
:(. Тут сразу несколько нарушений стайлгайда